### ISSUE TITLE
Include number of copies in card information

### ISSUE THREAD

**Issue by fredrikz** [Kamalisk/arkhamdb#188]
Some cards like https://arkhamdb.com/card/01098 only come as a one-of. It would be ideal if arkhamdb.com could also list this information, as it would help when checking that your weakness collection has all cards etc.

**Comment by oldbie** [comment]
I agree that this would be helpful. I noticed that cardgamedb.com has a quantity for each card. I was trying to figure out how many Emergency Caches were in the Core Set earlier, and couldn't seem to locate this information on ArkhamDB.

**Comment by erikoliver** [comment]
FWIW, you can download the cardgamedb.com data and cross-correlate the quantities relatively easily. I've started doing that to analyze scenarios (e.g. average shroud, etc.)
